mruby-mspec-support/mrblib/mruby-mspec-support.rb

```diff
@@ -27,10 +22,6 @@ def readlines(path)
       end
       a
     end
-
-    def split(path)
-      [File.dirname(path), File.basename(path)]
-    end
```

Comment:
Yes, `File.split` doesn't use on ruby/mspec.
But, ruby/spec is using it. (e.g. [require_spec](https://github.com/ruby/spec/blob/9daa86110d22a5bf38b391bd3bfe663bc38fe502/core/kernel/shared/require.rb#L10) and of cause [File.split](https://github.com/ruby/spec/blob/master/core/file/split_spec.rb))

How do you think in this case?